src/components/cc-heptapod-info/cc-heptapod-info.js

```diff
@@ -67,39 +70,48 @@ export class CcHeptapodInfo extends LitElement {
           ${i18n('cc-heptapod-info.description')}
         </div>
 
-        ${!this.error && !isNotUsed ? html`
-          <div class="pricing">
-            <div class="pricing-item">
-              <div class="pricing-item-value ${classMap({ skeleton })}">${statistics.privateActiveUsers}</div>
-              <div>${i18n('cc-heptapod-info.private-active-users-description')}</div>
-            </div>
-            <div class="pricing-item">
-              <div class="pricing-item-value ${classMap({ skeleton })}">${statistics.publicActiveUsers}</div>
-              <div>${i18n('cc-heptapod-info.public-active-users-description')}</div>
-            </div>
-            <div class="pricing-item">
-              <div class="pricing-item-value ${classMap({ skeleton })}">${i18n('cc-heptapod-info.storage-bytes', statistics)}</div>
-              <div>${i18n('cc-heptapod-info.storage-description')}</div>
-            </div>
-            <div class="pricing-item">
-              <div class="pricing-item-value ${classMap({ skeleton })}">${i18n('cc-heptapod-info.price-value', statistics)}</div>
-              <div>${i18n('cc-heptapod-info.price-description')}</div>
-            </div>
-          </div>
+        ${this.state.type === 'error' ? html`
+          <cc-notice intent="warning" message="${i18n('cc-heptapod-info.error-loading')}"></cc-notice>
         ` : ''}
 
-        ${!this.error && isNotUsed ? html`
+        ${this.state.type === 'not-used' ? html`
           <div class="no-statistics">${i18n('cc-heptapod-info.not-in-use')}</div>
         ` : ''}
 
-        ${this.error ? html`
-          <cc-notice intent="warning" message="${i18n('cc-heptapod-info.error-loading')}"></cc-notice>
-        ` : ''}
+        ${this._isLoadedOrLoading(this.state) ? this._renderStatistics(statistics, skeleton) : ''}
```

Comment:
nitpick: like my other comment, I think it would benefit from inlining the subrender directly here as it's not that much code and we use it only once.